src/App.css

```diff
@@ -1,6 +1,6 @@
 :root {
-  --will-display: none;
-  --remove-button: none;
+  --will-display: block;
+  --remove-button: block;
```

Comment:
why "block" is assigned to a variable name "--remove-button"?

Reply:
Previously remove button was displayed only on hover 
Now removed the hover functionality so block is assigned.

Reply:
Variables are used to describe the value. using --remove-button variable name for block value doesn't make any sense. Please update that.

Reply:
Oh okay 
Removed all the unwanted CSS variables